plugins/azure/applicationGateway/agHttpsListenerOnly.js

```diff
@@ -0,0 +1,53 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Application Gateway HTTPS Listener',
+    category: 'Application Gateway',
+    domain: 'Network Access Control',
+    severity: 'Medium',
+    description: 'Ensures that Application Gateway is not configured to use any non-HTTPS listener.',
```

Comment:
```suggestion
    description: 'Ensures that Application Gateway is configured to use only HTTPS listeners.',
```
I think phrasing it this way adds more clarity